### ISSUE TITLE
Segfault in gks_drv_socket

### ISSUE THREAD

**Issue by blegat** [jheinen/GR.jl#422]
The documentation build of Polyhedra.jl is currently failing with a segfault in libGR.so, see
https://github.com/JuliaPolyhedra/Polyhedra.jl/pull/265/checks?check_run_id=3124874873#step:5:35
The last build that was succeeding can be found here:
https://github.com/JuliaPolyhedra/Polyhedra.jl/runs/3002336981
Any idea what could be the issue ?

**Comment by jheinen** [comment]
As you are using GR (as a dependency) an a headless system, could you please `export GKS_WSTYPE=100` (or `ENV["GKS_WSTYPE"]="100"` in your build script(s)?

**Comment by blegat** [comment]
Thanks, do you think it would be possible to print an error showing this message instead of the segfault? This seems to be a very common pitfall people will run into

**Comment by jheinen** [comment]
It seems that setting the above environment didn't fix the problem. Could you provide a MWE?

**Comment by blegat** [comment]
Using `GKS_WSTYPE: 100` in the YAML config file of Github action gave `ERROR: LoadError: SystemError: opening file "/tmp/jl_yWGWNY.svg": No such file or directory`:
https://github.com/JuliaPolyhedra/Polyhedra.jl/runs/3160056474?check_suite_focus=true#step:5:14
Using `GKSwstype: nul` worked

**Comment by jheinen** [comment]
Makes sense. In `Plots.jl`, you will have to set `GKSwstype` to `nul` or `100`, as `GKS_WSTYPE` is used internally.
